tools/ed64romconfig.c

```diff
@@ -170,6 +309,14 @@ int main(int argc, char *argv[])
 	fwrite(cart_id, 1, CART_ID_SIZE, write_file);
 	fseek(write_file, VERSION_OFFSET, SEEK_SET);
 	fwrite(&config, 1, VERSION_SIZE, write_file);
+	fseek(write_file, CONTROLLERTYPE1_OFFSET, SEEK_SET);
+	fwrite(&controller_type1, 1, 1, write_file);
```

Comment:
["When in Rome, do as the Romans do"](https://tommcfarlin.com/coding-standards-and-guidelines/)

A `CONTROLLERTYPE_SIZE` constant is not strictly necessary, but I do think it is "better".